File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--programming-against-events-of-a-namedrange-control_4.cs
        private void namedRange1_SelectionChange(Microsoft.Office.Interop.Excel.Range Target)
        {
            this.namedRange2.Value2 = "The SelectionChange event occurred.";
            this.namedRange2.AddComment("SelectionChange always occurs before BeforeDoubleClick.");
            this.namedRange2.Font.Color = System.Drawing.ColorTranslator.ToOle(System.Drawing.Color.Black);
        }

File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--programming-against-events-of-a-namedrange-control_4.vb
    Private Sub NamedRange1_SelectionChange(ByVal Target As Microsoft.Office.Interop.Excel.Range) _
        Handles NamedRange1.SelectionChange

        With Me.NamedRange2
            .Value2 = "The SelectionChange event occurred."
            .AddComment("SelectionChange always occurs before BeforeDoubleClick.")
            .Font.Color = System.Drawing.ColorTranslator.ToOle(System.Drawing.Color.Black)
        End With
    End Sub